.github/workflows/test.yml

```diff
@@ -0,0 +1,67 @@
+name: Test
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+permissions:
+  contents: read
```

Comment:
Is this permission needed?

Reply:
I am not 100% sure about how GitLab actions work here. But my understanding is, that the default would be `write-all`, which probably is too broad?

- https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs
- https://dev.to/azu/how-to-set-github-actions-s-permissions-hln

Therefore I would suggest to leave it at `read` for `contents`, as this is all, this job needs.

Reply:
We can keep the default permission. It's fine :)

Reply:
Curious: What is the default of this repository? Wouldn't `write-all` not open the door for arbitrary actions from jobs? 🤔

Reply:
I don't know what's the default config tbh, but I guess just removing the permissions config would be fine.